src/NHibernate.Test/NHSpecificTest/GH1759/ColumnsAndFormulasFixture.cs

```diff
@@ -0,0 +1,209 @@
+				// So using ISession.Delete instead of ISession.CreateQuery("delete...
+				session.Delete("from User");
+				session.Delete("from Group");
+
+				transaction.Commit();
+			}
+		}
+
+		[Test]
+		public void CheckManyToMany()
+		{
+			using (var session = OpenSession())
+			using (var tx = session.BeginTransaction())
+			{
+				var bob = session.Query<User>().Single(u => u.Name == "Bob" && u.Org == "Eg");
+				var sally = session.Query<User>().Single(u => u.Name == "Sally" && u.Org == "Eg");
+				var salesperson = session.Query<Group>().Single(u => u.Name == "Salesperson" && u.Org == "Eg");
+				var manager = session.Query<Group>().Single(u => u.Name == "Manager" && u.Org == "Eg");
+
+				Assert.That(
+					bob.Groups,
+					Has.Count.EqualTo(1)
+					   .And.One.EqualTo(salesperson));
+				Assert.That(
+					sally.Groups,
+					Has.Count.EqualTo(2)
+					   .And.One.EqualTo(salesperson)
+					   .And.One.EqualTo(manager));
+				Assert.That(
+					salesperson.Users,
+					Has.Count.EqualTo(2)
+					   .And.One.EqualTo(bob)
+					   .And.One.EqualTo(sally));
+				Assert.That(
+					manager.Users,
+					Has.Count.EqualTo(1)
+					   .And.One.EqualTo(sally));
+
+				tx.Commit();
+			}
+		}
+
+		[Test]
+		public void CheckProperty()
+		{
+			using (var session = OpenSession())
+			using (var tx = session.BeginTransaction())
+			{
+				var bob = session.Query<User>().Single(u => u.Name == "Bob" && u.Org == "Eg");
+				var sally = session.Query<User>().Single(u => u.Name == "Sally" && u.Org == "Eg");
+
+				Assert.That(
+					bob.Hiring,
+					Has.Property(nameof(DateSpan.Date1)).EqualTo(DateTime.Parse("2001-01-01"))
+					   .And.Property(nameof(DateSpan.Date2)).EqualTo(bob.DepartureDate));
+				Assert.That(
+					sally.Hiring,
+					Has.Property(nameof(DateSpan.Date1)).EqualTo(DateTime.Parse("2006-01-08"))
+					   .And.Property(nameof(DateSpan.Date2)).EqualTo(sally.DepartureDate));
+
+				sally.DepartureDate = DateTime.Today;
+
+				tx.Commit();
+			}
+
+			using (var session = OpenSession())
+			using (var tx = session.BeginTransaction())
+			{
+				var sally = session.Query<User>().Single(u => u.Name == "Sally" && u.Org == "Eg");
+				Assert.That(sally.DepartureDate, Is.Not.Null);
+				Assert.That(
+					sally.Hiring,
+					Has.Property(nameof(DateSpan.Date1)).EqualTo(DateTime.Parse("2006-01-08"))
+					   .And.Property(nameof(DateSpan.Date2)).EqualTo(sally.DepartureDate));
+
+				tx.Commit();
+			}
+		}
+
+		[Test]
+		public void CheckElement()
+		{
+			using (var session = OpenSession())
+			using (var tx = session.BeginTransaction())
+			{
+				var salesperson = session.Query<Group>().Single(u => u.Name == "Salesperson" && u.Org == "Eg");
+
+				Assert.That(
+					salesperson.DateSpans,
+					Has.Count.GreaterThan(0)
+					   .And.All.Property(nameof(DateSpan.Date2)).EqualTo(DateTime.Parse("2000-12-31").AddYears(2)));
+
+				tx.Commit();
+			}
+		}
+
+		[Test, Ignore("Needs an unrelated additional simple fix in OneToManyPersister")]
```

Comment:
The fix is #1760 and is merged since long, but enabling this test has been forgotten in #1808. Although that is a hbm test only, I am enabling it here by the way. This PR is still quite related, and I do not fix enabling this test deserves a separated PR.